packages/client/package.json

```
@@ -14,6 +14,7 @@
   "exports": {
     "browser": "./dist/streamr-client.web.js",
     "default": {
+      "types": "./dist/types/src/index.d.ts",
```

Maybe would be ok to use `exports.d.ts` directly:

```suggestion
      "types": "./dist/types/src/exports.d.ts",
```